### PR TITLE
Removes TEG from roundstart engine rotation once again

### PR DESCRIPTION
**Pull request by N3D6**
# Document the changes in your pull request

tessa asked me to do this and i agreed because this engine is shit can we stop adding it back plz

atmos engines do not belong as the roundstart engine. normal engineers are not atmos techs.

# Why is this good for the game?
TEG is a garbage engine that isn't even designed for engineers, it's designed for atmospheric technicians. If you don't have someone guiding you the engine is almost impossible to set up with the guide and the roundstart setup usually needs modifications just to even run.

This engine is nowhere near beginner friendly, as it requires you to have atmospheric knowledge (pipes clogging, ht pipes etc) that beginners are simply not going to know. More often than not, I see many engineers (myself included) completely ignoring the TEG and going to do solars instead.

# Testing
haven't tested it but it's a two line change and i'm not going to

# Spriting

# Wiki Documentation

# Changelog

:cl:  
rscdel: removes teg from roundstart rotation (stop adding it back dear god)
/:cl:

### CODE REVIEW

**Comment by missatessatessy** [comment]
The engine either needs to be reworked so that engineers with 0 Atmos knowledge are able to set it up following a basic guide, or it needs to be removed and be reconsidered as an atmosian project type engine.

Tldr I agree with the change so engineer mains who don't understand atmos don't have to keep suffering

**Comment by cowbot92** [comment]
Would rather see it reworked than removed, it keeps being added because the children yearn for it. The cycle of adding/removing is strange..

**Comment by wonderinghost** [comment]
I just reworked the TEG wiki page ffs

**Comment by missatessatessy** [comment]
> I just reworked the TEG wiki page ffs

When you say just, do you mean the February version? This is the one I used to setup teg and I was unable to set it up properly as an engineer main.

**Comment by N3D6** [comment]
> Would rather see it reworked than removed, it keeps being added because the children yearn for it. The cycle of adding/removing is strange..

please find me the children that yearn for it

**Comment by wonderinghost** [comment]
> > I just reworked the TEG wiki page ffs
> 
> When you say just, do you mean the February version? This is the one I used to setup teg and I was unable to set it up properly as an engineer main.

Pretty sure that didn't have images. Now it has one (blame mord for less)

**Review by ToasterBiome — APPROVED** [review]

**Comment by iamthesilly** [comment]
> Would rather see it reworked than removed, it keeps being added because the children yearn for it. The cycle of adding/removing is strange..

The problem with the TEG as a main engine is its source of heat. Using a high-temperature plasma burn means the engineers need to have the atmospherics experience required to set up and maintain that, as well as find workarounds to all the problems it causes such as the pumps failing from the excessive pressure.

Reworking it would require giving it a different heat source and rebalancing it around that, and the only viable option I can think of that wouldn't cause the same problems is the nuclear reactor, at which point it should just be a part of the nuclear reactor setup instead of its own thing.

**Comment by wonderinghost** [comment]
The teg is going to be in that weird limbo state until it breaks and no one works on it or it's deletion. Probably in the same state as the experimentor, pods, and lawyers/clerks

This current iteration was to address the main problems I saw when it got canned, no fuel and a more confusing wiki page than my edit. If you want to rework this engine then it probably requires quirks to whenever we work on liquid physics.

But in it's current state it's still just a bad wiki page and setup. I brought this engine back in hope for the community to see it and figure out on how to improve on the previous. It wasn't my intention before to remove the cluwne spell, but pull it back from being a wizard spammable and figure out how to make it not ass for the people affected by the spell or condition while letting wizards that want the spell to still enjoy it.

**Comment by N3D6** [comment]
Closing this for now, will revisit if TEG improvements don’t end up merged

**Comment by N3D6** [comment]
pr to fix it was closed

oh brother this thing STINKS

**Comment by wonderinghost** [comment]
Wiki page had a massive glow up by @missatessatessy

**Comment by Moltijoe** [comment]
already approved by a HD, good to merge

**Comment by AMyriad** [comment]
it's for the best

**Comment by missatessatessy** [comment]
> pr to fix it was closed
> 
> oh brother this thing STINKS

I was literally only alerted to this pr today, I told you that this pr was good to close because the main problem with the Teg engine was resolved already. That was the guide was updated such that round start engineers were able to setup the engine successfully without much atmos knowledge.
This was not dependent on the teg fix pr as the goal of the fix pr would have had the same result as the wiki guide fix.

This should be reverted, because there is no good reason to keep it out of rotation other than the usual "it's boring" complaints.
